docs/topics/gradle.md

```diff
@@ -677,6 +626,31 @@ dependencies {
 </tab>
 </tabs>
 
+## Support for Gradle plugin variants
```

Comment:
I am no so sure this information is needed for the user here :thinking:

User neither could change variant selection, nor do something with this except providing additional information in the YT issue.

Probably, it would be better either not to mention it at all, or just leave variants names and how to check which variant was applied. What's new should be enough for this part.

Reply:
> I am no so sure this information is needed for the user here 🤔

I agree, that's why I asked you in DMs in Slack)

I'll delete it from here.